src/pages/[platform]/build-a-backend/auth/guest-access/index.mdx

```diff
@@ -0,0 +1,55 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Enable guest access',
+  description: 'Access services without needing to sign in.',
+  platforms: [
+    'javascript',
+    'react-native',
+    'flutter',
+    'swift',
+    'android',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+<InlineFilter filters={['flutter','swift','android']}>
```

Comment:
it looks like these two inline filters are the same? are they needed?

Reply:
the URLs are different for these platforms.